src/components/NavBar.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from 'react';
+import NavLinks from './NavLinks';
+
+export function NavBar({ user, lists, listPath }) {
+	const [isNavOpen, setIsNavOpen] = useState(false);
+
+	return (
+		<>
+			{/* DESKTOP MENU */}
+			<nav className="h-16 hidden lg:flex flex-row shadow-md justify-between items-center bg-offWhite text-darkPurple">
+				<div className="h-full flex items-center pl-12">
+					<h1 className="font-amiri text-2xl font-bold">Smart Shopping List</h1>
```

Comment:
I am not sure about including an h1 inside a nav tag or if it would be better to have an h2 as in the [example in the mdn nav docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) and each view include their own h1 ... what do you think?

Reply:
you are absolutely right, thanks for spotting this! I will change it for an h2 tag.